test/integration/src/main/java/io/pravega/test/integration/BookKeeperIntegrationTestBase.java

```diff
@@ -0,0 +1,77 @@
+/**
+ * Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.test.integration;
+
+import io.pravega.common.io.FileHelpers;
+import io.pravega.segmentstore.server.host.BookKeeperRunner;
+import io.pravega.segmentstore.server.store.ServiceBuilderConfig;
+import io.pravega.segmentstore.server.store.ServiceConfig;
+import io.pravega.segmentstore.server.store.StreamSegmentStoreTestBase;
+import java.io.File;
+import java.nio.file.Files;
+import lombok.AccessLevel;
+import lombok.Getter;
+
+/**
+ * Base class for any StreamSegmentStore Integration Test that uses BookKeeper.
+ */
+abstract class BookKeeperIntegrationTestBase extends StreamSegmentStoreTestBase {
```

Comment:
This class is a base test class for `FileSystemIntegrationTest` so I am not sure it's a good idea to copy it over from Pravega Core repository.  

Is it possible to reference it from the corresponding test jar instead?